### PR TITLE
React router set up

### LINKED ISSUE #7: Set Routing
## User story

As an user i want to navigate quick to the next page to make a desicion.

## ToDos

- [x] Read Doc of React Routing
- [x] Implement paths
- [x] Send to review

### PR DESCRIPTION
**Pull request by Matt-1985**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,16 +1,41 @@
...
+          <Route path="/desicion">
+            <DesicionPage />
+          </Route>
+          <Route path="/select">
+            <SelectBoxScreen />
+          </Route>
+          <Route path="/choice">
+            <DesicionScreenLazy />
+          </Route>
+          <Route path="/detailDesicion">
```
Comment (AlexisRoe) [inline client/src/App.js]: paths are always in lowerCase
```suggestion
          <Route path="/detaildesicion">
```

**Review by ncbecker — APPROVED** [review]
Lgtm - router implementation well done 👍 👍

File: client/src/pages/SelectBoxScreen.js [hunk 2]
```diff
@@ -1,28 +1,49 @@
...
+        console.log("shit");
+    }
+  };
+
   return (
-    <PageContainer>
+    <>
+      {select1 && console.log({ select1 })}
+      {select2 && console.log({ select2 })}
+      {select3 && console.log({ select3 })}
```
Comment (AlexisRoe) [inline client/src/pages/SelectBoxScreen.js]: remove such experiments before pushing 😉
```suggestion
```

**Review by AlexisRoe — COMMENTED** [review]
🎉 Routing juhu 🎉

File: client/src/pages/SelectBoxScreen.js (hunk 2 above)
Comment (Matt-1985) [reply client/src/pages/SelectBoxScreen.js]: Perfect! thanks for the hint... :) i will change that

File: client/src/pages/SelectBoxScreen.js [hunk 3]
```diff
@@ -1,28 +1,46 @@
...
-  margin-bottom: 15vh;
   padding: 15%;
-  gap: 4vh;
+  gap: 35px;
 `;
 
 export default function SelectBoxScreen() {
+  const [select1, setSelect1] = useState(null);
+  const [select2, setSelect2] = useState(null);
+  const [select3, setSelect3] = useState(null);
```
Comment (phlgr) [inline client/src/pages/SelectBoxScreen.js:18]: Not good naming

File: client/src/pages/StartPage.js [hunk 4]
```diff
@@ -1,17 +1,22 @@
...
+  justify-content: center;
+  grid-area: content;
   place-items: center;
 `;
 
 export default function StartPage() {
   return (
     <Div>
-      <FirstActionButton />
+      <Link to="/desicion" label="desicion">
```
Comment (phlgr) [inline client/src/pages/StartPage.js:17]: Spelling of `desicion`. Change to `decision`

**Pull request merged**